workflows/cluster_plots/__main__.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+
+import luigi
+from google.cloud import storage
+
+from birdclef.knn_labels import (
+    compute_embedding_2d,
+    get_knn_labels,
+    get_label_agreement,
+    get_subset_pdf,
+    plot_distances,
+    plot_embedding,
+    write_plots_to_disk,
+)
+from birdclef.utils import get_spark
+from workflows.utils.pull import Pull
+
+from .plotting import ClusterPlotAllTasks
+
+
+class ClusterPlotTaskWrapper(luigi.WrapperTask):
+    output_path = luigi.Parameter()
+    list_species = luigi.Parameter()
+
+    def requires(self):
+        pull_data = Pull(
+            input_path="data/processed/birdclef-2022/birdnet-embeddings-with-neighbors",
+            output_path="../data/processed/birdclef-2022/birdnet-embeddings-with-neighbors",
+            parallelism=os.cpu_count(),
+        )
+        yield pull_data
+
+        yield ClusterPlotAllTasks(
+            output_path=self.output_path,
+            total_cnt=len(self.list_species),
+        )
+        # yield cluster_plot_all_task
+
+
+def list_species():
```

Comment:
This should have an argument to to the path, and get called in the main section appropriately.

Also `json.loads(Path(path_to_json).read_text())` is personally my go-to for reading in json.